src/EditorFeatures/Core/GoToDefinition/AbstractGoToCommandHandler`2.cs

```diff
@@ -253,6 +252,9 @@ private Task DelayAsync(CancellationToken cancellationToken)
     private async Task FindResultsAsync(
         IFindUsagesContext findContext, Document document, int position, CancellationToken cancellationToken)
     {
+        // Ensure that we relinquish the thread so that the caller can proceed with their work.
+        await Task.Yield().ConfigureAwait(false);
```

Comment:
The standardized API for this is:

```
await TaskScheduler.Default.SwitchTo(true);
```

Reply:
You can also call `ConfigureAwait` directly and not rely on it appearing as an extension method.